sealir-tutorials/conda_environment.yml

```diff
@@ -0,0 +1,16 @@
+channels:
+  - defaults
+dependencies:
+  - python=3.12
+  - llvmlite=0.44.*
+  - mlir=17.0.*
+  - pytest
+  - networkx
+  - numpy
+  - nbconvert
+  - ipykernel
```

Comment:
does it make sense to add jupyterlab to this list, so launching jupyter from inside the environment works?  ipykernel is sufficient to use the env as a jupyter kernel, but this assumes you have a Jupyter UI installed somewhere else.

Reply:
added jupyter